Feature/infobar

jcaliksygic opened the pull request.

jcaliksygic committed:
SYUIInfobar

- customizable view
- example

jcaliksygic committed:
Merge branch 'develop' of github.com:Sygic/sygic-ui-kit-ios into feature/infobar

jcaliksygic committed:
design by maros holly

Kraci commented on SygicUIKit/Classes/Extensions/UIStackViewExtensions.swift:
```diff
@@ -23,6 +23,7 @@
 import Foundation
 
 public extension UIStackView {
+    /// Removes all arranded items and removes them from view hierarchy although.
```
Mozno by som tu metodu lepsie pomenoval, nech ked mi vybehne v auto complete viem co robi. Plus ten riadok oldViews je zbytocny

jcaliksygic replied:
ako by si ju lepsie pomenoval? jj oldViews vyhodim... v ObjC bol problem ked si editoval pole ktore si prave iteroval...

Kraci commented on SygicUIKit/Classes/Views/ActionButton/SYUIActionButton.swift:
```diff
@@ -56,12 +56,18 @@ public protocol SYUIActionButtonProperties {
     /// In case of alert.
     case alert
+    /// Primary action
+    case primary13
```
Co je primary13 a error13? Som to moc z toho nazvu nepochopil plus ten komentor k tomu je rovnaky ako primary. Napadla ma prva nejaka velkost, ale podla kodu sa tam s velkostou nic nedeje.

jcaliksygic replied:
nevedel som ako to pomenovat... je to modra farbicka co pouziva maros v dizajnoch... takze predpoklad je ze tato farbicka neskor nahradi staru primary... kym ale neprejdeme plne na nove dizajnisy, musime ju nejako odlisit a primary13 (iOS13) mi prislo lepsie ako primaryNew alebo newPrimary

Kraci commented on SygicUIKit/Classes/Views/Infobar/SYUIInfobar.swift:
```diff
@@ -0,0 +1,194 @@
+import UIKit
+
+/// TODO: neviem ci SYUIInfobarView nie je lepsie
+public class SYUIInfobar: UIView {
```
Je to rovnake ako SYUICompass, tak bud pisme bez view alebo s view. Ale vacsinou vsade piseme ci sa jedna o view alebo controller, ked ten subor hladas je to podla mna presnejsie ked vies ze je to view. Tak dajme kludne aj sem aj ku compassu aj vsade view.

jcaliksygic committed:
SYUIInfobarLabel items

jcaliksygic committed:
Icons assets catalog

jcaliksygic committed:
Merge branch 'develop' of github.com:Sygic/sygic-ui-kit-ios into feature/infobar

jcaliksygic committed:
private icons bundle

jcaliksygic committed:
Merge branch 'develop' of github.com:Sygic/sygic-ui-kit-ios into feature/infobar

jcaliksygic committed:
Merge branch 'develop' of github.com:Sygic/sygic-ui-kit-ios into feature/infobar

jcaliksygic committed:
Infobar example update

jcaliksygic committed:
Default system fonts

- deleted Inter-UI fonts
- deleted SygicUI nested font bundle
- SYUIFont types enum
-

Kraci commented on SygicUIKit/Classes/Views/Infobar/SYUIInfobarView.swift:
```diff
@@ -0,0 +1,183 @@
+    }
+    
+    /// Primary row items. Appearence is restricted by maxItemCount.
+    public var items: [UIView] = [] {
```
```suggestion
    public var items = [UIView]() {
```

Kraci commented on SygicUIKit/Classes/Views/Infobar/SYUIInfobarView.swift:
```diff
@@ -0,0 +1,183 @@
+    }
+    
+    /// Secondary row items with separator between each item. Appearence is restricted by maxItemCount.
+    public var secondaryItems: [UIView] = [] {
```
```suggestion
    public var secondaryItems = [UIView]() {
```

Kraci commented on SygicUIKit/Classes/Views/Infobar/SYUIInfobarView.swift:
```diff
@@ -0,0 +1,183 @@
+        stackView.alignment = .center
+        stackView.layoutMargins = UIEdgeInsets(top: edgeMargin, left: edgeMargin, bottom: edgeMargin, right: edgeMargin)
+        stackView.spacing = edgeMargin
+        stackView.isLayoutMarginsRelativeArrangement = true
```
tu by sa ku stackView uplne hodila ta closure definicia ^^

Kraci replied:
aj k tym ostatnym, ale akoze kludne to mozes nehat aj takto

Kraci reviewed (approved):
👍👍👍

jcaliksygic committed:
Merge branch 'develop' of github.com:Sygic/sygic-ui-kit-ios into feature/infobar

jcaliksygic committed:
new color assets

jcaliksygic committed:
infobar night background color

jcaliksygic committed:
infobar secondary items centering

jcaliksygic committed:
infobar layout refactor

jcaliksygic committed:
Update SygicUIKit/Classes/Views/Infobar/SYUIInfobarView.swift

Co-Authored-By: Jakub Kracina <kracina.jakub@gmail.com>

jcaliksygic committed:
Update SygicUIKit/Classes/Views/Infobar/SYUIInfobarView.swift

Co-Authored-By: Jakub Kracina <kracina.jakub@gmail.com>

jcaliksygic committed:
Merge pull request #27 from Sygic/feature/default-system-fonts

Default system fonts
